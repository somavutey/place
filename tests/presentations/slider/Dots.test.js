import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import Dots from "../../../components/presentations/Slider/Dots";

it("Dots", () => {
  const wrapper = shallow(<Dots />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
