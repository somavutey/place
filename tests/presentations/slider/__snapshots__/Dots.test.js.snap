// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dots 1`] = `
<div
  css={
    Array [
      "
      position: absolute;
      bottom: 25px;
      width: 100%;
      display: flex;
      align-items: center;
      justify-content: center;
    ",
    ]
  }
/>
`;
